src/Services/EncounterService.php

```
@@ -709,18 +709,16 @@ public function getPatientEncounterListWithCategories($pid)
     /**
      * Returns the sensitivity level for the encounter matching the patient and encounter identifier.
-     *
-     * @param  $pid          The legacy identifier of particular patient
-     * @param  $encounter_id The identifier of a particular encounter
-     * @return string         sensitivity_level of first row of encounter data
+     * Returns null when no matching encounter row exists.
      */
-    public function getSensitivity($pid, $encounter_id)
+    public function getSensitivity(mixed $pid, mixed $encounter_id): ?string
     {
         $encounterResult = $this->search(['pid' => $pid, 'eid' => $encounter_id], $options = ['limit' => '1']);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$options'. (undefined)

(UnusedLocalVariable)